### ISSUE TITLE
Snake Eyes Issue

### ISSUE THREAD

**Issue by JDFlashffxi** [EdenServer/community#1240]
Sometimes after using Sneak Eyes it wont roll a 1 instead will try to roll normally which then breaks the roll and giving a message that the roll is now 0 and no amount of rerolling raises the number til i start a whole new roll. It also removes all the rolls from the party members im with when it breaks. so far ive only had it happen when im in a party.

**Comment by jmcmorris** [comment]
If you can give exact steps to reproduce that'd be helpful in tracking down what this issue is. Thanks!

**Comment by Mikokiko** [comment]
![Miko_2019 03 10_153551](https://user-images.githubusercontent.com/43910014/54091576-e4b82580-4357-11e9-8bc9-04582cba950d.png)

I just rolled normaly up to 8, for test pupose, then I used snake eyes, and it automatically act liek it rolls a 6, then you get the message it wolled 0. 

Snake eye is suppose to be rolling a 1 with a 10% chance to give you 11 (additional 10% per additionnal merit), not a guarantee bust. It also does not get removed after you roll. It shouldnt last the full duration it's on, but be removed as soon as you roll after using the job ability.
